src/Cpanel/ClientActions.php

```diff
@@ -60,15 +61,21 @@ public function mergeCpanelAndCFDomains()
             $cpanelDomainList = array_merge($cpanelDomainList, $getCpanelDomains['parked_domains']);
         }
 
+        $Punycode = new Punycode();
```

Comment:
The fix is dependent on `GET /zone?name` endpoint. `cloudflare-plugin-backend` only has generic calls. Max we can do is move the getter/setter there but I think it'd be better if each plugin has it separately unless we move`GET /zone?name` to `cloudflare-plugin-backend`